docs/use_cases/personalized_search.md

```diff
@@ -75,21 +73,21 @@ user_preference_weight = 0.3
 biased_query_embedding = query_weight * query_embedding + user_preference_weight * user_preference_vector
 ```
 
-In this code example, we convert a search query into a vector using an [open-source, pretrained BERT model from Hugging Face](https://huggingface.co/bert-base-uncased) (You can also try this out online using the link). We also have a user preference vector, which would be usually based on a user's past clicks or choices. We then arithmetically "add" the query vector and the user preference vector to create a new query vector that reflects both the user input as well as user preferences.
+In this code example, we convert a search query into a vector using an [open-source, pretrained BERT model from Hugging Face](https://huggingface.co/bert-base-uncased) (You can try this out online yourself by following the link). We also have a user preference vector, which would be usually be based on a user's past clicks or choices. We then arithmetically "add" the query vector and the user preference vector to create a new query vector that reflects both the user input and user preferences.
```

Comment:
instead of "which would be usually be based"
say "which would usually be based"